app/panel/components/Subscribe.jsx

```diff
@@ -11,7 +11,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { openSupporterPage } from '../utils/msg';
```

Comment:
@zarembsky let's rename `openSupporterPage` to `openSubscriptionPage` everywhere